flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/KeyedStream.scala

```diff
@@ -183,7 +183,10 @@ class KeyedStream[T, K](javaStream: KeyedJavaStream[T, K]) extends DataStream[T]
    * Creates a new [[DataStream]] by folding the elements of this DataStream
    * using an associative fold function and an initial value. An independent 
    * aggregate is kept per key.
+   *
+   * @deprecated will be removed in a future version
    */
+  @Deprecated
```

Comment:
Do you propose to only have the reason in the annotation, or in both annotation and docs?